openms-wiki/about.md

```diff
@@ -0,0 +1,14 @@
+
+About
+=====
```

Comment:
There is an About OpenMS: https://staging-openms.readthedocs.io/en/staging/docs/index.html

Can you merge these contents?

Reply:
don't know how....

Reply:
delete this file.